frontend/src/container/GridGraphLayout/Graph/FullView/Label.tsx

```diff
@@ -0,0 +1,26 @@
+import { LabelContainer } from './styles';
+import { getAbbreviatedLabel } from './utils';
+
+function Label({
```

Comment:
this are component for the table right can we move this one to `TableRender/Label` ?

Reply:
Okay